apps/web/src/app/(dashboard)/documents/delete-document-dialog.tsx

```diff
@@ -50,6 +50,13 @@ export const DeleteDocumentDialog = ({
     },
   });
 
+  useEffect(() => {
+    if (open) {
+      setInputValue('');
+      setIsDeleteEnabled(false);
```

Comment:
This line won't allow users to delete draft documents. See the main comment here https://github.com/documenso/documenso/pull/762#issuecomment-1862367650 🙏

Reply:
Updated! Now when deleting draft documents the button not disabled.